interfaces/repo_test.go

```diff
@@ -813,6 +845,78 @@ func (s *RepositorySuite) TestResolveDisconnectMatrixNoSnaps(c *C) {
 	}
 }
 
+// All the was to resolve a 'snap disconnect' between two snaps.
```

Comment:
What does: `All the was` mean? Typo for "ways"?

Reply:
Yes, it was supposed to be "ways", let me check for other cases and fix that